examples/transformer/custom-transformer/PyFunc-Transformer.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# PyFunc Model + Transformer Example\n",
     "\n",
-    "This notebook demonstrates how to deploy a Python function based model and a custom transformer. This type of model is useful as user would be able to define their own logic inside the model as long as it satisfy contract given in `merlin.PyFuncModel`. If the pre/post-processing steps could be implemented in Python, it's encouraged to write them in the PyFunc model code instead of separating them into another transformer.accepts a tensor input. The transformer has preprocessing step that allows the user to send a raw image data and convert it to a tensor input.accepts a tensor input. The transformer has preprocessing step that allows the user to send a raw image data and convert it to a tensor input.\n",
+    "This notebook demonstrates how to deploy a Python function based model and a custom transformer. This type of model is useful as user would be able to define their own logic inside the model as long as it satisfy contract given in `merlin.PyFuncModel`. If the pre/post-processing steps could be implemented in Python, it's encouraged to write them in the PyFunc model code instead of separating them into another transformer. The transformer has preprocessing step that allows the user to send a raw image data and convert it to a tensor input. The transformer has preprocessing step that allows the user to send a raw image data and convert it to a tensor input.\n",
```

Comment:
`The transformer has preprocessing step that allows the user to send a raw image data and convert it to a tensor input` this sentence is repeated

Reply:
udpated